orderbook/Cargo.toml

```diff
@@ -10,6 +10,8 @@ model = { path = "../model" }
 primitive-types = "0.7"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+shared-arguments = { path = "../shared-arguments" }
+structopt = { version = "0.3" }
```

Comment:
Maybe arguments should
```
pub use structopt::StructOpt;
```

To guarantee the versions are compatible?

Reply:
@e00E In case this gets merged before you see this comment.

Reply:
In this case I think that is better because we only use structopt in main.rs. Will make this change.

Reply:
Sadly it doesn't work because of how the structopt derive macro is set up. It expects users to directly depend on structopt.